consensus/src/epoch_manager.rs

```diff
@@ -308,14 +308,17 @@ impl EpochManager {
             Ok(()) => { /* Expected */ }
             Err(Error::ValidatorKeyNotFound(error)) => {
                 error!(
+                    epoch = epoch,
                     error = error,
-                    "Unable to find the validator consensus key during initialize",
+                    "Unable to find the validator consensus key during initialize() and will not participate in consensus.\
+                     Will continue syncing until the validator consensus key matches in a future epoch."
                 );
             }
             Err(Error::ValidatorNotInSet(error)) => {
                 error!(
+                    epoch = epoch,
                     error = error,
-                    "The validator is not in the current validator set",
+                    "The validator is not in the current validator set for the epoch and will not participate in consensus.",
                 );
             }
             Err(error) => panic!(
```

Comment:
I recommend we eliminate the panic / expect statement altogether ... if consensus can make progress, let it, who cares what happened to safety rules.... that's safety rules problem.

Reply:
Interesting.. I don't know if I have enough context on how consensus operates here, but assuming it'll just continue to do its thing, this makes sense to me. I'll wait for the others to comment.